### PR TITLE
Renamed PowerPlatformExtensionContext to WebExtensionContext

### PR DESCRIPTION
**Pull request by BidishaMS**

### CODE REVIEW

File: src/web/client/WebExtensionContext.ts [hunk 1]
```diff
@@ -52,74 +52,74 @@ class WebExtensionContext {
         currentSchemaVersion: ""
     };
 
-    public getPowerPlatformExtensionContext() {
-        return this.powerPlatformExtensionContext;
+    public getWebExtensionContext() {
```
Comment (tyaginidhi) [inline src/web/client/WebExtensionContext.ts:55]: we can actually make this a getter function with name context. In any case we are accessing all of these as WebExtensionContext.<func>. Or, any better suggestions here?

File: src/web/client/WebExtensionContext.ts [hunk 2]
```diff
@@ -52,74 +52,74 @@ class WebExtensionContext {
...
+        return this.webExtensionContext;
     }
 
-    public async setPowerPlatformExtensionContext(entityName: string, entityId: string, queryParamsMap: Map<string, string>) {
+    public setWebExtensionContext(entityName: string, entityId: string, queryParamsMap: Map<string, string>) {
```
Comment (tyaginidhi) [inline src/web/client/WebExtensionContext.ts:59]: this can be a setter function too

**Review by tyaginidhi — APPROVED** [review]

File: src/web/client/WebExtensionContext.ts (hunk 1 above)
Comment (BidishaMS) [reply src/web/client/WebExtensionContext.ts:55]: As discussed created a task https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3116420

File: src/web/client/WebExtensionContext.ts (hunk 2 above)
Comment (BidishaMS) [reply src/web/client/WebExtensionContext.ts:59]: https://dynamicscrm.visualstudio.com/OneCRM/_workitems/edit/3116420

**Pull request merged**